docs/guides/security/authentication/active_directory_authentication.md

```diff
@@ -134,6 +122,19 @@ for an Active Directory user.
 administrator@ad.company.local:*:1450400500:1450400513:Administrator:/home/administrator@ad.company.local:/bin/bash
 ```
 
+!!! Note 
+
+    `getent` get entries from Name Service Switch libraries (NSS), it means that, contrary to `passwd` or `dig` for example, it will ask for responses in different database, including `/etc/hosts` for `getent hosts` or from `sssd` in the `getent passwd` case.
```

Comment:
```suggestion
    `getent` get entries from Name Service Switch libraries (NSS), it means that, contrary to `passwd` or `dig` for example, it will ask for responses in different databases, including `/etc/hosts` for `getent hosts` or from `sssd` in the `getent passwd` case.
```